openc3/bin/openc3cli

```diff
@@ -141,18 +141,21 @@ def migrate(args)
 
   # Overwrite plugin.txt with specified targets
   plugin = File.open('plugin.txt', 'w')
-  plugin.puts "TARGET #{target_name}"
-
+  FileUtils.mkdir 'targets'
```

Comment:
This should probably have a force so that it doesn't fail if already there.

Reply:
It can't fail because we just generated a new plugin. This is only intended to be run once.